pkgs/tools/X11/dex/default.nix

```diff
@@ -1,18 +1,19 @@
-{ stdenv, fetchFromGitHub, python3 }:
+{ stdenv, fetchFromGitHub, python3, sphinx }:
```

Comment:
no need to add `sphinx` here, you can reference it as `python3.pkgs.sphinx`, so the argument `python3` is enough.

Reply:
Ah, I just copied it because I saw some other packages doing it that way, but fixed now.